README.md

```diff
@@ -1,13 +1,13 @@
-WebExtensions / Chrome browser add-on for linking a browser to the KeePassRPC KeePass plugin.
+WebExtensions (Mozilla Firefox) / Google Chrome (Chromium) browser add-on for linking browser to KeePassRPC KeePass plugin.
 
 Build requirements:
 
-# node
-# npm
-# the usual hundreds of node modules installed by the above
+* node
+* npm or yarn
```

Comment:
Have you tested this? Or is this other tool always guaranteed to be a 100% working drop-in replacement for npm?